daemon/api_interfaces_test.go

```diff
@@ -1185,3 +1186,30 @@ func (s *interfacesSuite) TestInterfacesModern(c *check.C) {
 		"type":        "sync",
 	})
 }
+
+func (s *interfacesSuite) TestInterfacesAllDefaultDocURL(c *check.C) {
+	_ = s.daemon(c)
+
+	req, err := http.NewRequest("GET", "/v2/interfaces?select=all&doc=true", nil)
+	c.Assert(err, check.IsNil)
+	rec := httptest.NewRecorder()
+	s.req(c, req, nil).ServeHTTP(rec, req)
+	c.Check(rec.Code, check.Equals, 200)
+	var body map[string]interface{}
+	err = json.Unmarshal(rec.Body.Bytes(), &body)
+	c.Check(err, check.IsNil)
+
+	if result, ok := body["result"].([]interface{}); ok {
+		c.Assert(len(result), check.Equals, 190, check.Commentf("New interface detected, please ensure it is documented at https://snapcraft.io/docs/<name>-interface and increment the count"))
```

Comment:
I think this is an annoying place to do this check as it's too far away from interfaces,  what you could do is export the number somehow from interfaces or interfaces/builtin

Reply:
https://github.com/canonical/snapd/pull/14273/commits/6d92b6b057d6b19616dd91168641de4d7f7e7ea6